tests/func/test_delta.py

```diff
@@ -96,6 +96,76 @@ def create_delta_dataset(ds_name):
     create_delta_dataset(ds_name)
 
 
+def test_delta_update_unsafe(test_session, tmp_dir, tmp_path):
+    catalog = test_session.catalog
+
+    default_namespace_name = catalog.metastore.default_namespace_name
+    default_project_name = catalog.metastore.default_project_name
+
+    starting_ds_name = "starting_ds"
+    merge_ds_name = "merge_ds"
+
+    dependency_ds_name = (
+        f"{default_namespace_name}.{default_project_name}.{starting_ds_name}"
+    )
+    dependency_ds_merge_name = (
```

Comment:
is it only to use `(_get_dependencies(catalog, ds_name, "1.0.1")` ? (quite unfortunate tbh)